app/models/interview_question.rb

```diff
@@ -2,4 +2,26 @@ class InterviewQuestion < ActiveRecord::Base
   belongs_to :interview
   validates :interview_id, presence: true
   validates :question_id, presence: true
+
+  def self.get_interview_question_number(interview, question)
+    interview_question_number = interview.interview_questions.index(question) + 1
+    interview_question_number
+  end
+
+  def self.num_question(question, num)
+    number = num + 1
+    number = number.to_s + 'c' unless question.comment.blank?
+    number
+  end
+
+  def self.interview_question_next(interview, question)
+    next_question = interview.interview_questions.where("id > ?", question.id).first
+    next_question
+  end
```

Comment:
1. no need in ```next_question``` 
2. this should be an instance method:
```ruby
def self.interview_question_next
   interview.interview.interview_questions.where("id > ?", id).first
end
```
3. give names in more natural manner like ```next_interview_question```